src/control/jobs/CustomExportJob.cpp

```diff
@@ -47,11 +47,11 @@ void CustomExportJob::addFilterToDialog()
 {
 	XOJ_CHECK_TYPE(CustomExportJob);
 
-	addFileFilterToDialog(EXPORT_PDF, "*.pdf");
-	addFileFilterToDialog(EXPORT_PDF_NOBG, "*.pdf");
-	addFileFilterToDialog(EXPORT_PNG, "*.png");
-	addFileFilterToDialog(EXPORT_PNG_NOBG, "*.png");
-	addFileFilterToDialog(EXPORT_XOJ, "*.xoj");
+	// Runs on every filter inside the filters map
```

Comment:
Yes, much better than before!
Only one thing: C++11 also supports foreach on maps:
https://stackoverflow.com/questions/6963894/how-to-use-range-based-for-loop-with-stdmap

for (auto& filter : filters)